src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
Interestingly, while the [accessbility training](http://webaim.org/presentations/2017/training/ARIAHTML5.pdf) we attended suggested using the "dialog" value for aria-haspopup, I can't find mention of that on a cursory search (only true/false).  In fact, most people are saying not to use that attribute at all unless it's a sub-menu or menu that's popping up.  [More](https://www.levelaccess.com/expanding-the-use-of-the-aria-haspopup-property/)